geopolars/src/geoseries.rs

```diff
@@ -30,8 +30,52 @@ pub enum TransformOrigin {
     Point(Point),
 }
 
+pub enum BroadcastableParameter<'a, T> {
```

Comment:
I'm still wrapping my head around lifetimes, so I might have a couple dumb questions.

Here we need a lifetime on the enum because the enum takes a reference to the series right? And otherwise the pointer from the enum to the series could outlive the memory contained by the series?

If a function takes a `&Series` as an input parameter directly, would it still need a lifetime? Or no because the series wouldn't be stored inside an object like an enum?